com.hperrin.StreamOverlay.yml

```diff
@@ -0,0 +1,55 @@
+id: com.hperrin.StreamOverlay
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node22
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  # Needed for opening, saving config files.
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+build-options:
+  append-path: /usr/lib/sdk/node22/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: stream-overlay
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/stream-overlay/flatpak-node/cache
+        npm_config_cache: /run/build/stream-overlay/flatpak-node/npm-cache
+        npm_config_nodedir: /usr/lib/sdk/node22
+        npm_config_offline: "true"
+    build-commands:
+      # Install app wrapper
+      - install -Dm755 -t /app/bin/ ./run.sh
+      - install -D -m644 -t /app/share/applications ./source/com.hperrin.StreamOverlay.desktop
+      - install -D -m644 -t /app/share/metainfo ./source/com.hperrin.StreamOverlay.metainfo.xml
+      - install -D -m644 -t /app/share/mime/packages/ ./source/com.hperrin.StreamOverlay.mime.xml
+      - install -D -m644 -T ./source/app/assets/512x512.png /app/share/icons/hicolor/512x512/apps/com.hperrin.StreamOverlay.png
+      - install -D -m644 -T ./source/app/assets/logo.svg /app/share/icons/hicolor/scalable/apps/com.hperrin.StreamOverlay.svg
+      - mkdir /app/main/
+      - mv ./archive/* /app/main/
+    sources:
+      - type: git
+        url: https://github.com/hperrin/stream-overlay.git
+        commit: f2183c0381c2809f47a6b5d2dba3994f0e08e9fa
+        dest: source
+      - type: archive
+        url: https://github.com/hperrin/stream-overlay/releases/download/v6.0.0/stream-overlay-linux.tar.gz
+        sha256: 2225e4e813a13a7c109a0a083adc3efa2a558daade81ddc9e42423c12f177370
+        dest: archive
```

Comment:
why not build from source?

Reply:
If you can kinda hold my hand in getting it to work, I'd be glad to do it so it builds from source. I think that would be better anyway.

Reply:
>  but I had no luck getting it to work.

What was the issue?

Reply:
I don't really remember. That was like a year ago and I gave up. Is it possible to run `npm install` in the build process?

Reply:
and look at how other things are built https://github.com/search?q=org%3Aflathub%20%22npm%20install%22&type=code

you need to add the node SDK too https://github.com/flathub/org.freedesktop.Sdk.Extension.node22

Reply:
Ok, I'm very confused now. Tutanota has that command in their script, but it looks like they don't actually build from source:

https://github.com/flathub/com.tutanota.Tutanota/blob/7a7ecf400507bc1ea16f6927661ca736be354ec1/com.tutanota.Tutanota.json#L60

Reply:
Ok, it looks like they're just using a script to update that manifest after every release. So it's not actually building from source.

Reply:
This is really confusing me and it doesn't seem like there's any benefit to building from source if the dependency file needs to be manually updated each time.

Reply:
I haven't tested it at all on arm. I'd prefer not to have to support it until I properly test it.

Reply:
Is this something that's required to get it submitted?